automation/canary-prod-test.sh

```diff
@@ -17,7 +16,16 @@ else
     echo "Starting canary test in Production"
 fi
 
-JSON_CREDS=`docker run --rm -e VAULT_TOKEN=$VAULT_TOKEN -e VAULT_ADDR=https://clotho.broadinstitute.org:8200 broadinstitute/dsde-toolbox vault read -format=json secret/dsde/firecloud/prod/common/canary/firecloud-account.json | jq '.data'`
+DOCKER_ARGS=(
+  "run"
+  "--rm"
+  "-v ${HOME}/.config/gcloud:/root/.config/gcloud"
+  "google/cloud-sdk"
+)
+
+SECRET_ACCESS_ACCOUNT=jenkins-firecloud@broad-dsp-techops.iam.gserviceaccount.com
+# Expand the array of args and pass them to `docker`
+JSON_CREDS=$(docker ${DOCKER_ARGS[*]} /bin/bash -c "gcloud config set account ${SECRET_ACCESS_ACCOUNT} && gcloud secrets versions access latest --project broad-dsde-dev --secret firecloud-sa")
```

Comment:
Because you are mounting the gcloud root directory from the host OS, running this command could mutate the active gcloud user for all jobs not just this one